internal/store/endpoint.go

```diff
@@ -150,6 +151,27 @@ func endpointMetricFamilies(allowAnnotationsList, allowLabelsList []string) []ge
 				}
 			}),
 		),
+		*generator.NewFamilyGenerator(
+			"kube_endpoint_port",
```

Comment:
Could we name the metric `kube_endpoint_ports` to be consistent with `kube_endpoint_labels` and `kube_endpoint_annotations`?